### PR TITLE
Kraken update2

### PR DESCRIPTION
**Pull request by nekrut**
kraken actually works now. added kraken-mpa-report. still need to add tests

### CODE REVIEW

**Comment by davebx** [comment]
:+1:

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+${show_zeros}
+${header_line}
+
+> "$output_report"
+
+
+	]]>
+    </command>
+    <inputs>
+      <repeat name="classifications" title="Classifications" min="1">
```
Comment (bgruening) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: Can we use multiple="True" here?

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ 4. A rank code, indicating (U)nclassified, (D)omain, (K)ingdom, (P)hylum, (C)lass, (O)rder, (F)amily, (G)enus, or (S)pecies. All other ranks are simply filled with a dash.
+ 5. NCBI taxonomy ID
+ 6. Indented scientific name
+
+The scientific names are indented using spaces, according to the tree
+structure specified by the taxonomy.
+    ]]>
+    </help>
+    <expand macro="version_command" />
+    <expand macro="requirements" />
```
Comment (bgruening) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: I think `planemo` will complain here. Requirements and stdio and version_command should be at the top.

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml (hunk 1 above)
Comment (nekrut) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: we certainly should

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml (hunk 2 above)
Comment (nekrut) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: I'll fix that

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml (hunk 1 above)
Comment (nekrut) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: actually, I appears that we cannot as it triggers multiple jobs instead

Comment (bgruening) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: @nekrut why is this so? It shouldn’t ... iteration over the `multiple=True` should be equivalent with repeat, afaik.

Comment (nekrut) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: yes, this is how I did it in the latest commit

**Comment by bgruening** [comment]
I'm going to merge this, but this tool really needs some tests.

**Pull request merged**